src/jsonLanguageTypes.ts

```diff
@@ -128,6 +128,11 @@ export interface DocumentLanguageSettings {
 	 * The severity of reported trailing commas. If not set, trailing commas will be reported as errors.
 	 */
 	trailingCommas?: SeverityLevel;
+
+	/**
+	 * The severity of schema validation. If se to 'ignore', schema validation will be skipped. 
```

Comment:
We should also define what the default is. I suggest, for backward compatibility, 'warning'.

Also there's a `t` missing in `set`

Reply:
I have improved the description :D